src/widgets/infinite-hits/infinite-hits.tsx

```diff
@@ -46,9 +46,24 @@ export type InfiniteHitsTemplates = {
 };
 
 export type InfiniteHitsRendererWidgetParams = {
+  /**
+   * Escapes HTML entities from hits string values.
+   *
+   * @default `true`
+   */
   escapeHTML: boolean;
-  transformItems: (items: any[]) => any[];
+  /**
+   * Enable the button to load previous results.
+   *
+   * @default `false`
+   */
   showPrevious: boolean;
+  /**
+   * Receives the items, and is called before displaying them.
+   * Should return a new array with the same shape as the original array.
+   * Useful for mapping over the items to transform, and remove or reorder them.
+   */
+  transformItems: (items: any[]) => any[];
```

Comment:
Can we see this kind of comments on editor like.. on tooltip when hovering? or they're just commented there?

Reply:
They work when the import is coming from the TypeScript file itself (`src/widgets/infinite-hits/infinite-hits.ts`). That means that you can read them from the test files and from the stories if using the direct file (not `instantsearch.widgets.infiniteHits`).

It doesn't work when loaded from other files because we don't export the types yet.

<details>
  <summary>From a test</summary>

![image](https://user-images.githubusercontent.com/6137112/63173166-8cd8c580-c03f-11e9-89d7-04200b9f05bd.png)

</details>

<details>
  <summary>From a story (import modified locally</summary>

![image](https://user-images.githubusercontent.com/6137112/63172908-07edac00-c03f-11e9-89a0-e68244655fc1.png)

</details>

Reply:
Thanks for the screen shot!
I've checked and saw it working on `infinite-hits-test.ts`.
Something I don't understand is,

![image](https://user-images.githubusercontent.com/499898/63173944-3cfafe00-c041-11e9-9143-d401cc86b97d.png)

This is from `infinite-hits.tsx` and why doesn't it work? It looks like it should work.

Reply:
I think this is a bug in the TypeScript VS Code extension because the definition is showing up for `cssClasses`, which is using a renaming with the `actualName: otherName` syntax.

![image](https://user-images.githubusercontent.com/6137112/63174516-5ea8b500-c042-11e9-8c2c-340bdaf39019.png)

Reply:
You're right. I've tested a few times and it only works for the renamed keys.